collector/src/main/java/io/prometheus/jmx/JmxScraper.java

```diff
@@ -14,6 +14,7 @@
 import java.util.logging.Logger;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+
```

Comment:
removed (left over from experimentation, injected by Eclipse to separate java. and other imports